components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -299,21 +341,25 @@ export class Panel extends React.Component<Props, State> {
         >
           {
             publisher && publisher.publisher_key
-            ? <WalletPanel
-              id={'wallet-panel'}
-              platform={publisher.provider as Provider}
-              publisherName={publisher.name}
-              publisherImg={faviconUrl}
-              monthlyAmount={10}
-              isVerified={publisher.verified}
-              tipsEnabled={true}
-              includeInAuto={!publisher.excluded}
-              attentionScore={(publisher.percentage || 0).toString()}
-              onToggleTips={this.doNothing}
-              donationAction={this.showDonateToSiteDetail}
-              onAmountChange={this.doNothing}
-              onIncludeInAuto={this.switchAutoContribute}
-            />
+            ? <>
+              <WalletPanel
+                id={'wallet-panel'}
+                platform={publisher.provider as Provider}
+                publisherName={publisher.name}
+                publisherImg={faviconUrl}
+                monthlyAmount={defaultContribution}
+                isVerified={publisher.verified}
+                tipsEnabled={true}
+                includeInAuto={!publisher.excluded}
+                attentionScore={(publisher.percentage || 0).toString()}
+                onToggleTips={this.doNothing}
+                donationAmounts={this.generateAmounts()}
+                donationAction={this.showDonateToSiteDetail}
+                onAmountChange={this.onContributionAmountChange.bind(this, publisher.publisher_key)}
+                onIncludeInAuto={this.switchAutoContribute}
+              />
+              <div style={{ height: '48px' }}></div>
```

Comment:
This is a place holder that fills the space that would be taken up by the "Enable tips on action" component. This will be reworked once that feature is supported.